zenodo/modules/records/serializers/schemas/datacite.py

```diff
@@ -444,6 +446,41 @@ def inject_type(c):
 
         return items
 
+    def get_descriptions(self, obj):
+        """Get descriptions."""
+        items = []
```

Comment:
To avoid duplication you can call `items = super(DataCiteSchema, self).get_description(obj)`  and then just keep the added `method`-related code and return `items`